toobigtoignore/issf
# Made catfish a staff user

## What
Gave the testing user, catfish, superadmin and staff permissions


## Why
To enable testing of staff-only functionality, such as adding new tips, or disabling tweeting.


## Testing
1. Load the new database dump (scripts/postgres_setup.sh)
2. Log in as catfish
3. Click "catfish" in the top right
4. You should now see that 3 staff-only options have appeared

![](https://i.bootleg.technology/lx5LTSl0Z8.png)
